fem/restriction.cpp

```diff
@@ -684,6 +690,20 @@ H1FaceRestriction::H1FaceRestriction(const FiniteElementSpace &fes,
      gather_indices(nf*dof)
 {
    if (nf==0) { return; }
+
+#ifdef MFEM_USE_MPI
+
+   // If the underlying finite element space is parallel, ensure the face neighbor information is generated.
+   const ParFiniteElementSpace *pfes = dynamic_cast<const ParFiniteElementSpace*>
+                                       (&fes);
+
+   if (pfes != NULL)
+   {
+      pfes->GetParMesh()->ExchangeFaceNbrData();
+   }
```

Comment:
```suggestion
   // If the underlying finite element space is parallel, ensure the face
   // neighbor information is generated.
   if (const ParFiniteElementSpace *pfes
       = dynamic_cast<const ParFiniteElementSpace*>(&fes))
   {
      pfes->GetParMesh()->ExchangeFaceNbrData();
   }
```

Minor nitpicks: wrapping comment at 80 columns, restricting scope of `pfes` to the `if` statement.

Reply:
I admit, I'm lazy and do whatever the style checker tells me to do. 😄